code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -432,6 +432,20 @@
 /datum/reagent/mutationtoxin/proc/mutate(mob/living/carbon/human/H)
 	if(QDELETED(H))
 		return
+	if (race=="random")
+		to_chat(H, mutationtext)
+		H.set_species(pick(/datum/species/jelly/slime,
```

Comment:
Instead of listing these twice in two separate procs, just add a single list var to the reagent

Reply:
fixed